src/node.cc

```diff
@@ -1739,6 +1739,10 @@ static void GetInternalBinding(const FunctionCallbackInfo<Value>& args) {
     // internalBinding('natives_hash')
     exports = Object::New(env->isolate());
     DefineJavaScriptHash(env, exports);
+  } else if (!strcmp(*module_v, "v8_extras")) {
+    // override `exports` return value, GetExtrasBindingObject is persistent.
+    args.GetReturnValue().Set(env->context()->GetExtrasBindingObject());
```

Comment:
Not sure it's a great idea to expose the extras binding object this way. @hashseed ... thoughts?

Reply:
I would like to note that this is exposed through `internalBinding`, which was previously fully internal but made to be exposed through `--expose-internals` to allow easier testing.

Reply:
Well, now that I look at it more, it's not actually clear where this is used anywhere by this PR. Am I missing it?

Reply:
@jasnell I added it to expose IsReadableStream but yeah nothing uses it yet.